internal/resources/azure/sql_elasticpool.go

```diff
@@ -0,0 +1,42 @@
+package azure
```

Comment:
Do we need this file - it doesn't look like we use it?

Reply:
👍 cheers, have deleted